libregexp.c

```diff
@@ -864,17 +868,43 @@ static int re_parse_char_class(REParseState *s, const uint8_t **pp)
     cr_init(cr, s->opaque, lre_realloc);
     p = *pp;
     p++;    /* skip '[' */
+
+    if (s->unicode_sets) {
+        static const char verboten[] =
```

Comment:
Why is it german? 😅

Reply:
Or Dutch ;-)

Reply:
> Why is it german? 😅

To make it sound more affirmative? Using the French *interdit* would definitely sound less absolute.

As explained in this quote is attributed to Sir Winston Churchill:

In England, everything is permitted except what is forbidden.
In Germany, everything is forbidden except what is permitted.
In France, everything is allowed, even what is prohibited.
In the USSR, everything is prohibited, even what is permitted.

Reply:
> Or Dutch ;-)

No, it would be *verboden* in dutch, unless a local dialect in the northern part of Holland is closer to German.

Reply:
The dialect around here is in fact very close to Low German but really the reason I used "verboten" is because I'm a big fan of German as spoken (well...) in that one scene from the movie Der Untergang :)

Reply:
> but really the reason I used "verboten" is because I'm a big fan of German

Okay 😂 Nice one

Reply:
Dann können wir hier auf Deutsch diskutieren.  Kannst to auch Saúl ?

Reply:
No chance 😅

Reply:
Tut mir Leid... let's switch back to the lingua franca :)

Reply:
Na gut, akzeptiert xD